atmo/templates/atmo/jobs/detail.html

```diff
@@ -163,6 +164,36 @@
         {% endfor %}
         </ul>
       </div>
+      <div role="tabpanel" class="tab-pane" id="runs">
+        {% if spark_job.has_never_run %}
+          <p>No job run history yet.</p>
+        {% else %}
+          <table class="table table-striped">
+            <thead>
+              <tr>
+                <th>Jobflow ID</th>
+                <th>EMR version</th>
+                <th>Last status</th>
+                <th>Scheduled</th>
+                <th>Ran</th>
+                <th>Terminated</th>
+              </tr>
+            </thead>
+            <tbody>
+              {% for run in spark_job.runs.all %}
+              <tr>
+                <td>{{ run.jobflow_id }}</a></td>
```

Comment:
@robhudson The region is configured in the `settings.AWS_CONFIG['AWS_REGION']` setting, so let's use that for the value. I made a comment in the review for this.